operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md

```diff
@@ -0,0 +1,196 @@
+# Update default Air-Cooled BMC and Leaf switch SNMP credentials
+
+This procedure updates the default credentials used when new air-cooled hardware is discovered for the first time. This includes the default Redfish credentials used for new air-cooled NodeBMCs and Slingshot switch BMCs (RouterBMCs), and SNMP credentials for new management leaf switches.
+
+**Important:** After this procedure is completed **going forward all future air-cooled hardware** added to the system will be assumed to be already configured with the new global default credential when getting added to the system.
+
+**Note:** This procedure will not update the Redfish or SNMP credentials for existing air-cooled devices. To change the credentials on existing air-cooled hardware follow the [Change Air-Cooled Node BMC Credentials](Change_Air-Cooled_Node_BMC_Credentials.md) and [Change SMNP credentials on Leaf switches](Change_SMNP_credentials_on_Leaf_switches.md) procedures. 
+
+## Prerequisites
+
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+-   **Limitation:** The default global credentials used for liquid-cooled BMCs in the [Change Cray EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md) procedure needs to be the same as the one used in this procedure for air-cooled BMCs river hardware. 
```

Comment:
New section:

### Limitations

The default global credentials used for liquid-cooled BMCs in the [Change Cray EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md) procedure needs to be the same as the one used in this procedure for air-cooled BMCs river hardware.

Reply:
Should be resolved fixed now.